src/applications/facility-locator/components/ProviderServiceDescription.jsx

```diff
@@ -1,14 +1,20 @@
 import React from 'react';
-import { facilityTypes } from '../config';
+import { facilityTypes, ccUrgentCareLabels } from '../config';
 
-const providerName = query => {
+const providerName = (query, posCodes) => {
   let name;
   switch (query.facilityType) {
     case 'cc_pharmacy':
       name = facilityTypes.cc_pharmacy.toUpperCase();
       break;
     case 'urgent_care':
-      name = facilityTypes.urgent_care.toUpperCase();
+      if (posCodes && posCodes === 17) {
+        name = ccUrgentCareLabels.WalkIn;
+      } else if (posCodes && posCodes === 20) {
+        name = ccUrgentCareLabels.UrgentCare;
```

Comment:
The `posCodes &&` isn't necessary.

I'd recommend changing the magic numbers into constants that more clearly say what it is they mean.